lib/rtsp/session/manager.ex

```diff
@@ -106,4 +168,20 @@ defmodule Membrane.Protocol.RTSP.Session.Manager do
       {:error, :no_such_header} -> {:ok, state}
     end
   end
+
+  defp handle_nonce(%Response{} = response, state) do
```

Comment:
This function isn't exactly `handle_nonce` it's more like extract authentication spec. Please consider changing the name.